pymc/distributions/simulator.py

```diff
@@ -83,9 +83,9 @@ class Simulator(NoDistribution):
         be passed by order, in which case the keyword argument ``params`` is
         ignored. Alternatively, each parameter can be passed by order after fn,
         ``param1, param2, ..., paramN``
```

Comment:
Can't comment further up. It looks like `unnamed_params` is missing. I think it should be something like

```
*unnamed_params : list of TensorVariable
    description starts here
params : list of TensorVariable
    description extended here, plus mention that if both are present an error is thrown.
```

someone else will need to take over on the actual description though as I don't really know what to write. Is one preferred over the other, are there any potential behaviour differences? 

If they are both exactly the same, then the we can follow for example xarray's pattern where this situation is very common with kwargs, see for example: https://docs.xarray.dev/en/stable/generated/xarray.Dataset.rename.html

Reply:
It looks like they should behave identically. If the keyword argument isn't passed, then params just gets defined as the unnamed_params and then params is used throughout the rest of the file. I went ahead and wrote a description similar to xarray.